wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -73,6 +73,97 @@ body.sensei {
 			font-size: var(--wp--preset--font-size--heading-2);
 		}
 	}
+
+	section.wp-block-sensei-lms-course-outline {
+		> section.wp-block-sensei-lms-course-outline-module-bordered {
+			border: 1px solid var(--wp--preset--color--light-grey-1) !important;
+			border-radius: 2px;
+
+			header {
+				background-color: var(--wp--preset--color--light-grey-2) !important;
+				color: var(--wp--preset--color--charcoal-1) !important;
+				padding: var(--wp--preset--spacing--20);
+
+				h2 {
+					font-size: var(--wp--preset--font-size--heading-6);
+					font-weight: var(--wp--custom--button--typography--font-weight);
+				}
+
+				div.wp-block-sensei-lms-course-outline-module__progress-indicator {
+					border-radius: 2px;
+
+					&.completed {
+						color: var(--wp--custom--color--green-50) !important;
+					}
+
+					.wp-block-sensei-lms-course-outline-module__progress-indicator__text {
+						font-family: "SF Pro Text", sans-serif;
+						font-size: 11px;
+						font-weight: var(--wp--custom--button--typography--font-weight);
+						line-height: var(--wp--custom--body--huge--typography--line-height);
+						margin: 4px 8px;
+					}
+				}
+			}
+
+			.wp-block-sensei-lms-course-outline-module__lessons-title {
+				display: none;
+			}
+
+			.wp-block-sensei-lms-collapsible {
+				padding-top: 5px;
+				padding-bottom: 5px;
+			}
+		}
+
+		// every lesson item
+		.wp-block-sensei-lms-course-outline-lesson {
+			> span {
+				line-height: var(--wp--custom--body--short-text--typography--line-height);
+				padding: 15px var(--wp--preset--spacing--10);
```

Comment:
If lessons belong to a module, there is no gap between them, and the padding is `15px`. 
However, the parent div of these lessons will have a padding of `5px` on the top and bottom. 
Additionally, individual lessons that do not belong to any module should also have an extra `5px` of padding on the top and bottom.

Have tried several other approaches, but this one is the simplest and most straightforward. However, it has a different behavior on hover. I am considering removing the `5px` padding-top and padding-bottom from the parent div. What do you think?

| Hover on module lesson (Space on top and bottom) | Hover on none module lesson (Full) |
|-|-|
| ![image](https://github.com/WordPress/Learn/assets/18050944/afd687a8-f25d-45a8-9477-d1a54f726f24) | ![image](https://github.com/WordPress/Learn/assets/18050944/a9216942-06c1-40dc-8a17-e3dc12b42dee) |

Reply:
Yeah I reckon remove the 5px padding